ui/admin-portal/src/app/services/candidate-field.service.ts

```diff
@@ -27,12 +27,27 @@ import {
   checkForOverdue,
   TaskAssignment
 } from "../model/task-assignment";
+import {CandidateOpportunity} from "../model/candidate-opportunity";
+import {SavedList} from "../model/saved-list";
 
 @Injectable({
   providedIn: 'root'
 })
 export class CandidateFieldService {
 
+  // Dictates whether certain fields are available, e.g. Next Step and Added By, which require the
+  // source to be a submission list.
+  private candidateSource: CandidateSource;
+
```

Comment:
This "context" should be passed in with calls

Reply:
👍